corehq/motech/dhis2/migrations/0005_move_jsonapilog.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import unicode_literals
+from __future__ import absolute_import
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dhis2', '0004_auto_20171122_0827'),
+    ]
+
+    database_operations = [
+        migrations.AlterModelTable('JsonApiLog', 'motech_requestlog')
```

Comment:
I think that this will rename the database table, which:

a. (I believe) can block requests
b. it will cause any queries to fail between when this migration is run and when the new code is running
c. not easily revert-able

I don't know much about motech/dhis2 so not sure how much intermittent failures actually matter. What are your thoughts on just adding https://docs.djangoproject.com/en/1.11/ref/models/options/#db-table in the new model to avoid this?